### PR TITLE
INFRA:Initialize Unit Tests Project

### PR DESCRIPTION
**Pull request by ADILATIC**

### CODE REVIEW

**Review by glhays — COMMENTED** [review]
Project needs renamed to the standard naming conventions.
https://discord.com/channels/934130100008538142/1098701261882273792/1098704092504789015

**Review by glhays — COMMENTED** [review]
Remove Global Usings

File: Adi.Core.Api.Tests.Unit/Usings.cs [hunk 1]
```diff
@@ -0,0 +1 @@
+global using Xunit;
```
Comment (glhays) [inline Adi.Core.Api.Tests.Unit/Usings.cs:1]: 1. Remove Global Usings
2. Add copyright Header
// ---------------------------------------------------------------------------------- 
// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
// ----------------------------------------------------------------------------------

File: Adi.Core.Api.Tests.Unit/DeleteMe.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+namespace Adi.Core.Api.Tests.Unit
```
Comment (glhays) [inline Adi.Core.Api.Tests.Unit/DeleteMe.cs:1]: Add copyright Header
// ---------------------------------------------------------------------------------- 
// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
// ----------------------------------------------------------------------------------

This should be at the head of every file where possible, followed by the usings.

File: TestProject1/TestProject1.csproj [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (glhays) [inline TestProject1/TestProject1.csproj:1]: TestProject1 should not be part of this PR.

**Comment by hassanhabib** [comment]
@ADILATIC which issue does this closes?

File: Adi.Core.Api.Tests.Unit/DeleteMe.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void Test1()
```
Comment (hassanhabib) [inline Adi.Core.Api.Tests.Unit/DeleteMe.cs:6]: This should say `ShouldBeTrue`

Comment (hassanhabib) [reply Adi.Core.Api.Tests.Unit/DeleteMe.cs:6]: File should say `DeleteMe.cs` and the class should say `DeleteMe`99999ui
